.github/workflows/nodejs.yml

```diff
@@ -57,7 +57,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x, 18.x]
+        node-version: [14.x, 16.x, 18.x]
```

Comment:
I was talking to @ckniffen about this and he actually suggested keeping tests for 10.x and 12.x to ensure backwards compatibility for legacy systems.

Reply:
This breaks tests on my branch, so I'd prefer to do that as a separate ticket and get this merged

Reply:
Does 12.x also break tests? Leaving that in would at least keep it the same as it was previously.
If it does, then we should make a note to not publish the new version until we fix that.

Reply:
Yes jest does not support node 12.